ios/Converters/SQRDCard+RNReaderSDKAdditions.m

```diff
@@ -22,19 +22,22 @@
 
 @implementation SQRDCard (RNReaderSDKAdditions)
 
-- (NSMutableDictionary *)jsonDictionary
+- (NSDictionary *)jsonDictionary
 {
-    NSMutableDictionary *jsTransactionResult = [[NSMutableDictionary alloc] init];
-    jsTransactionResult[@"brand"] = [self getBrandString:self.brand];
-    jsTransactionResult[@"lastFourDigits"] = self.lastFourDigits;
-
-    return jsTransactionResult;
+    return @{
+        @"brand" : [self _stringForBrand:self.brand],
+        @"lastFourDigits" : self.lastFourDigits,
+        @"expirationMonth" : self.expirationMonth ?: [NSNull null],
```

Comment:
This is a behavior change -- is the presence of these NSNull objects going to affect integrators, e.g. if they are iterating over the dictionary and there are suddenly extra keys present that weren't in the previous version?

You could still do the mutable dictionary approach and only set the keys that are non-null, if you want.  Call `copy` on your dictionary at the end and return that, to make it immutable.

Reply:
Good suggestion, I'll use the mutable dictionary approach, and I don't think we need worry the type of dictionary too much given it's only used for plugin internally.